android/src/test/java/com/getcapacitor/ExampleUnitTest.java

```diff
@@ -0,0 +1,18 @@
+package com.getcapacitor;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
```

Comment:
can we delete this?

Reply:
Adjusted unit test, so that we have one for getDevicePPI().